python/ray/tests/test_runtime_env.py

```diff
@@ -115,7 +115,7 @@ def test_get_wheel_filename():
     for sys_platform in ["darwin", "linux", "win32"]:
         for py_version in ray_constants.RUNTIME_ENV_CONDA_PY_VERSIONS:
             # TODO(https://github.com/ray-project/ray/issues/31362)
-            if py_version == (3, 11) and sys_platform != "linux":
+            if py_version == (3) and sys_platform != "linux":
```

Comment:
What's the reason for this change?

Reply:
I saw you have a wheel for 3.11 but not for 3.3. So thought to only remove the 11. but ill just remove the whole case

Reply:
Oh sorry, (3,11) refers to 3.11